src/openapi/AccountLimit.yaml

```diff
@@ -17,7 +17,7 @@ info:
   # - initial Version of I_AccountLimit_Service
```

Comment:
Mit den Änderungen sollten wir hier mindestens auf Version 1.2 gehen, da das Passwort jetzt anders übertragen werden soll

Reply:
In der URL ja, aber die Version der Yaml Datei unterliegt einem anderen Schema